src/Helper/Fields/Field_Repeater.php

```diff
@@ -187,7 +187,7 @@ public function get_repeater_html( $value, $field ) {
 				if ( $sub_field instanceof GF_Field_Repeater ) {
 
 					/* Only recursively output if a value exists */
-					if ( isset( $item[ $sub_field->id ] ) ) {
+					if ( ! empty( $item[ $sub_field->id ] ) ) {
```

Comment:
Would it make more sense to use `! $field->empty_deep( $item[ $sub_field->id ] )` here, to standardise it like the other empty checks?